`azurerm_app_configuration_feature` - support `key` property (#21252)
hashicorp/terraform-provider-azurerm: website/docs/r/app_configuration_feature.html.markdown

* `enabled` - (Optional) The status of the App Configuration Feature. By default, this is set to false.

> * `key` - (Optional) The key of the App Configuration Feature. `%` character is not allowed in key. If not specified, `name` will be used as the key. Changing this forces a new resource to be created.

Reviewer: ```suggestion
* `key` - (Optional) The key of the App Configuration Feature. The value for `name` will be used if this is unspecified. Changing this forces a new resource to be created.
```

* `label` - (Optional) The label of the App Configuration Feature. Changing this forces a new resource to be created.

* `locked` - (Optional) Should this App Configuration Feature be Locked to prevent changes?

> * `name` - (Required) The name of the App Configuration Feature. `:` or `%` characters are not allowed in name. Changing this forces a new resource to be created.

Reviewer: ```suggestion
* `name` - (Required) The name of the App Configuration Feature. Changing this forces a new resource to be created.
```
